### PR TITLE
Pull/newsbeuter

### PR DESCRIPTION
**Pull request by afh**

### CODE REVIEW

**Comment by mikemcquaid** [comment]
Fails to build on 10.8: http://bot.brew.sh/job/Homebrew%20Pull%20Requests/21660/version=mountain_lion/testReport/junit/brew-test-bot/mountain_lion/install_newsbeuter/

Perhaps it needs C++11 (`needs :cxx11`, `ENV.cxx11`)?

**Comment by afh** [comment]
It sure does, I've updated this pull-request accordingly

File: Library/Formula/newsbeuter.rb [hunk 1]
```diff
@@ -1,56 +1,27 @@
...
+  depends_on "json-c"
+  depends_on "libstfl"
+  depends_on "sqlite"
 
-      ENV.append 'CXXFLAGS', '--std=c++11'
-    end
+  needs :cxx11
 
+  def install
+    ENV.cxx1
```
Comment (mikemcquaid) [inline Library/Formula/newsbeuter.rb]: typo here: `ENV.cxx11`

Comment (afh) [inline Library/Formula/newsbeuter.rb]: Fixed, thanks!

**Comment by tdsmith** [comment]
Thanks!

**Comment by afh** [comment]
Thanks for merging.

**Pull request closed without merging**